Master-Game/World/TransitionTile.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MasterGame.World
+{
+    class TransitionTile: BaseTile
+    {
+        private bool IsOpenForTransition;
```

Comment:
really this could just have the get set

Reply:
Got it, so you are saying I can just define it as:
private bool IsOpenForTransition = {get; set;}


Right?